blog/management/tests.py

```
@@ -0,0 +1,489 @@
+        )
+        self.client.force_login(self.staff_user)
+
+    def test_import_invalid_form_format(self):
+        """Test that parsed_import_data rejects malformed requests"""
+        # Create a non-JSON file
+        invalid_file = SimpleUploadedFile(
+            "test.txt", b"This is not JSON", content_type="text/plain"
+        )
+
+        rf = RequestFactory()
+        request = rf.post(
```

Ruff (0.8.2)

Local variable `request` is assigned to but never used

Remove assignment to unused variable `request`

(F841)